include/gmsh.tcc

```diff
@@ -135,52 +136,29 @@ void GMSH::read_elements(const std::string& filename) {
   std::cout << "Number of Elements: " << elements_.size() << '\n';
 }
 
-//! \brief Print Vertices Vector to text file
-//! \details to Check data entry correct
-void GMSH::output_vertices(const std::string& outputfilename) {
-
-  std::fstream outputfile;
-  outputfile.open(outputfilename, std::ios::out);
-
-  if (outputfile.is_open()) {
-
-    //! Iterate through vector and print
-    outputfile << tot_points_ << '\n';
-
-    for (const auto& point : vertices_) {
-      outputfile << point->coordinates().at(0) << '\t'
-                 << point->coordinates().at(1) << '\t'
-                 << point->coordinates().at(2) << '\n';
-    }
-    outputfile.close();
-  }
-
-  std::cout << "The output file for soil particles has been generated."
-            << "\n";
-}
-
 void GMSH::output_stresses(const std::string& outputfilename) {
 
   std::fstream outputfile;
   outputfile.open(outputfilename, std::ios::out);
 
-  double density = 22;
-  double K0 = 0.5;
+  const double density = 22;
+  const double K0 = 0.5;
```

Comment:
Naming convention! `K0` should be `k0`.
We have guideline: https://cpp.cb-geo.com/naming/variable_names.html, please follow these guidelines. Reviews are to find any coding errors/bugs, not to check style guidelines. `const` rule in the guideline is only for global variable.